app/src/main/java/org/wikipedia/language/LanguagesListViewModel.kt

```diff
@@ -0,0 +1,88 @@
+package org.wikipedia.language
+
+import android.content.Context
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.CoroutineExceptionHandler
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import org.apache.commons.lang3.StringUtils
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.dataclient.ServiceFactory
+import org.wikipedia.dataclient.mwapi.SiteMatrix
+import org.wikipedia.util.Resource
+import org.wikipedia.util.log.L
+import java.util.*
+
+class LanguagesListViewModel : ViewModel() {
+
+    val siteListData = MutableLiveData<Resource<List<SiteMatrix.SiteInfo>>>()
```

Comment:
Would it be possible to move public `val` after `private val`?

Reply:
sure